packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -36,6 +41,17 @@ export const LobbyPage = (props: LobbyPageProps): JSX.Element => {
   const callState = useSelector(getCallStatus);
   const inLobby = callState === 'InLobby';
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const participants = useSelector(getRemoteParticipants) ?? {};
+
+  const remoteParticipantsTrampoline = (): RemoteParticipantState[] | undefined => {
```

Comment:
Define this as a file-level free function. Currently, it's an anonymous function object redefined on each render.
Also, all these function definitions inline make the Component code harder to read.

Reply:
Also, move the `useSelector` call inside the trampoline so you need only one conditional compilation directive.